packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -345,10 +350,15 @@ export function handle_lazy_img(element) {
 	// templates.
 	if (!hydrating && element.loading === 'lazy') {
 		var src = element.src;
-		element.removeAttribute('loading');
+		// @ts-expect-error
+		element.__loading = null;
+		element.loading = 'eager';
 		element.removeAttribute('src');
 		requestAnimationFrame(() => {
-			element.loading = 'lazy';
+			// @ts-expect-error
+			if (element.__loading !== 'eager') {
```

Comment:
what's better about using __loading compared to what you had before?

Reply:
What I had before didn't work lol, if you remove loading, it becomes `eager` meaning that it never made it lazy in the original case.

Reply:
Should we use a symbol instead? Less risk of clashing with someone else.